### PR TITLE
SLC questions

### LINKED ISSUE #888: Change SLC Question char count to word count and improve their look
Currently, under the text boxes for SLC Questions we have a character counter, we should change this to a word counter and changed the wording at the top of the page to let users know they should have approximately 500 words total. 

Also, make the word counter text smaller and its lighter in color.

### PR DESCRIPTION
**Pull request by stevensonmichel**
- Updated variables' name
- edited the text at the top of the page to inform the user about the approximate total words
- included more words separators
- Overall, fixes Issue #888

### CODE REVIEW

File: app/static/js/slcNewProposal.js [hunk 1]
```diff
@@ -421,3 +423,26 @@ textareas.each(function(index, textarea) {
   });
   $(slcQuestionCharCount[index]).html($(textarea).val().length);
 });
```
Comment (AndersonStettner) [inline app/static/js/slcNewProposal.js]: Remove commented out code if it is not being used.

**Review by AndersonStettner — CHANGES_REQUESTED** [review]
You did not address the second change requested on the issue to make the counter smaller and lighter.

File: app/static/js/slcNewProposal.js (hunk 1 above)
Comment (azabeli) [reply app/static/js/slcNewProposal.js]: Resolved

**Comment by azabeli** [comment]
> You did not address the second change requested on the issue to make the counter smaller and lighter.

We resolved this!

**Review by AndersonStettner — CHANGES_REQUESTED** [review]
Replace punctuation with a space.

File: app/static/css/slcQuestionnaire.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.slcQuestionWordCounter{
+    font-size: 0.8em;
+    color: gray;
+}
+.slcQuestionWordCount{
```
Comment (BrianRamsay) [inline app/static/css/slcQuestionnaire.css]: the element with this class is inside of the other one, so the font-size and color are inherited and you don't need to specify them.

File: app/templates/serviceLearning/slcQuestionnaire.html [hunk 3]
```diff
@@ -1,48 +1,50 @@
 {%set title = "SLC Questionnaire"%}
 {% set isViewSlc = 'view' in request.path %}
 
+<link rel="stylesheet" href="{{url_for('static', filename ='css/slcQuestionnaire.css')}}">
```
Comment (BrianRamsay) [inline app/templates/serviceLearning/slcQuestionnaire.html:4]: This needs to be added into the styles block, not straight into the template.

File: app/static/js/slcNewProposal.js [hunk 4]
```diff
@@ -413,11 +413,22 @@ function enableSyllabusUploadFile() {
 }
 
 const textareas = $(".textarea");
-const slcQuestionCharCount = $(".slcQuestionCharCount");
+const slcQuestionWordCount = $(".slcQuestionWordCount")
```
Comment (BrianRamsay) [inline app/static/js/slcNewProposal.js]: We don't need the `.slcQuestionWordCount` class at all, actually. We can still get all of those elements with this selector: $(".slcQuestionWordCounter span")

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/templates/serviceLearning/slcQuestionnaire.html (hunk 3 above)
Comment (azabeli) [reply app/templates/serviceLearning/slcQuestionnaire.html:4]: resolved

File: app/static/js/slcNewProposal.js (hunk 4 above)
Comment (azabeli) [reply app/static/js/slcNewProposal.js]: resolved

File: app/static/css/slcQuestionnaire.css (hunk 2 above)
Comment (azabeli) [reply app/static/css/slcQuestionnaire.css]: resolved

File: app/static/js/slcNewProposal.js [hunk 5]
```diff
@@ -413,11 +413,26 @@ function enableSyllabusUploadFile() {
...
+
+function calculateCountWords(text){
+<<<<<<< HEAD
+  const words = text.split(/\s+/);
+=======
+  const words = text.split(/\s+|[;:,.~?!]+/);
+>>>>>>> refs/remotes/origin/slcQuestions
```
Comment (AndersonStettner) [inline app/static/js/slcNewProposal.js]: Resolve the merge conflict.

**Review by AndersonStettner — CHANGES_REQUESTED** [review]

File: app/static/js/slcNewProposal.js (hunk 5 above)
Comment (stevensonmichel) [reply app/static/js/slcNewProposal.js]: Merge conflict has been resolved

**Review by AndersonStettner — CHANGES_REQUESTED** [review]
The word count is not working on the page.

**Review by AndersonStettner — CHANGES_REQUESTED** [review]
"Words:" should not appear on the view page.

**Review by AndersonStettner — APPROVED** [review]

**Pull request merged**